src/ImpurityDensities/ThermalDiffusionLithiumDensity.jl

```diff
@@ -31,24 +31,49 @@ struct ThermalDiffusionLithiumDensity{T <: SSDFloat} <: AbstractImpurityDensity{
     lithium_diffusivity_in_germanium::T
 end
 
-function calculate_lithium_diffusivity_in_germanium(lithium_annealing_temperature::T)::T where {T <: SSDFloat}
+struct LithiumDiffusionBranch{T <: SSDFloat}
+    T_min::T
+    T_max::T
+    D0::T
+    H::T
+end
+
+struct LithiumDiffusionParameters{T <: SSDFloat}
+    lowT::LithiumDiffusionBranch{T}
+    highT::LithiumDiffusionBranch{T}
+end
+
+struct LithiumSaturationParameters{T <: SSDFloat}
+    a::T
+    b::T
+end
+
+struct ThermalDiffusionLithiumDensityParameters{T <: SSDFloat}
+    diffusion::LithiumDiffusionParameters{T}
+    saturation::LithiumSaturationParameters{T}
+end
+
+include("ThermalDiffusionLithiumDensityParameters.jl")
+
+function calculate_lithium_diffusivity_in_germanium(lithium_annealing_temperature::T,parameters::LithiumDiffusionParameters)::T where {T <: SSDFloat}
     # D0 [m^2*s^-1]
     # H [cal]
-    D0::T, H::T = ifelse(lithium_annealing_temperature <= 873, T.((2.5e-7, 11800)), T.((1.3e-7, 10700)))
+    D0::T, H::T = ifelse(lithium_annealing_temperature <= parameters.lowT.T_max, T.((parameters.lowT.D0, parameters.lowT.H)), T.((parameters.highT.D0, parameters.highT.H)))
```

Comment:
This could then iterate through the list, assuming it's sorted by increasing temperatures (maybe we would need to add a check that the `Tmax` of the previous entry doesn't overlap with `Tmin` of the following one);